Collibri/Models/Sections/SectionRepository.cs

```diff
@@ -0,0 +1,31 @@
+using Collibri.Models.DataHandling;
+
+namespace Collibri.Models.Sections
+{
+    public class SectionRepository : ISectionRepository
+    {
+        private readonly IDataHandler _dataHandler;
+
+        public SectionRepository(IDataHandler dataHandler)
+        {
+            _dataHandler = dataHandler;
+        }
+        
+        public Section? CreateSection(Section section, string roomName)
+        {
+            List<Section> sectionList = _dataHandler.GetAllItems<Section>(ModelType.Sections);
+            
+            foreach (var sections in sectionList)
+                if (sections.RoomId.Equals(section.RoomId) && sections.SectionName.Equals(section.SectionName))
+                    return null;
+            
```

Comment:
Toks vat pastebejimas. Žinau, kad vienam tam if/else sakiniui nereikia {}, bet kiek skaiciau ir is kitu girdejau, kad if/else geriau visad naudoti {}.

Reply:
okey pakeisiu